code/controllers/subsystem/SSblackbox.dm

```diff
@@ -327,7 +327,8 @@ SUBSYSTEM_DEF(blackbox)
 			"oxyloss" = L.getOxyLoss(),
 			"coord" = "[L.x], [L.y], [L.z]",
 			"server_id" = GLOB.configuration.system.instance_id,
-			"rid" = GLOB.round_id
+			"rid" = GLOB.round_id,
+			"last_words" = length(L.say_log) > 0 ? L.say_log[L.say_log.len] : null
```

Comment:
```suggestion
			"last_words" = length(L.say_log) > 0 ? L.say_log[length(L.say_log)] : null
```

Reply:
I wouldn't say this is necessary because we already null checked the list at the start of the ternary, and iirc reading .len is marginally faster than length(), but your discretion.